src/usr/bin/diffuse.py.in

```diff
@@ -860,6 +860,7 @@ class Preferences:
             [ 'List',
               [ 'Integer', 'tabs_default_panes', 2, _('Default panes'), 2, 16 ],
               [ 'Boolean', 'tabs_always_show', False, _('Always show the tab bar') ],
+              [ 'Boolean', 'tabs_reload_automatically', False, _('Reload external changes without prompt') ],
```

Comment:
I'm not sure this option is really necessary since you also added a way to reload all the files in just one click.

As for the binary files, you can create another PR with this change if you want.